index.d.ts

```diff
@@ -0,0 +1,116 @@
+// Type definitions for core-decorators.js 0.10
+// Project: https://github.com/jayphelps/core-decorators.js
+// Definitions by: Qubo <https://github.com/tkqubo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export interface ClassDecorator {
```

Comment:
Yup I agree with that, no point in doubling the number of type rules if they are provided to us already. Similarly the Console is defined already for us in `lib.dom.d.ts` so I've gone and removed that extraneous definition as well.